Kitodo-API/src/main/java/org/kitodo/config/Config.java

```diff
@@ -57,40 +53,94 @@ public static String getKitodoConfigDirectory() {
     public static String getParameter(String parameter) {
         try {
             return getConfig().getString(parameter);
-        } catch (RuntimeException e) {
+        } catch (NoSuchElementException e) {
             logger.error(e.getMessage(), e);
             return "No configuration found!";
```

Comment:
This one is not so good. It would be better to throw NoSuchElementException. Imagine a call like this:
`String tempDir = Config.getParameter("tempDir");` and now this is not defined:
`String imagePath = tempDir + "/temp.jpg"` // is now: No configuration found!/temp.jpg
Please remove the catch block

Reply:
Problem will be addressed in separated PR.